tests/framework/CodeInterfaceTests/tests

```diff
@@ -337,4 +337,11 @@
    UnorderedCsv = 'raven_running_raven_internal_models/testPointSet_dump.csv'
    rel_err = 0.0001
  [../]
+ 
+ [./SaphireNoExe]
+   type = 'RavenFramework'
+   input = 'test_saphire.xml'
+   test_interface_only = True
+   UnorderedCsv = 'testSaphire/saphirePrint.csv'
+ [../]
```

Comment:
please, create a Folder named "SAPHIRE" where you place this test...
It is going to help the user and (us) in the organization of the directories

Reply:
moved.